cli-config/src/config.rs

```diff
@@ -20,6 +20,8 @@ pub struct Config {
 
     #[serde(default)]
     pub address_labels: HashMap<String, String>,
+
+    pub commitment: String,
```

Comment:
I'm worried this'll cause the cli config to be overwritten with the default when moving between a cli that has/doesn't have this field.    Adding something like `#[serde(default = "singleGossip")]` should prevent this though

Reply:
Oh, good call, thx! Looks like `#[serde(default)]` is sufficient, due to default implementation https://github.com/solana-labs/solana/pull/14684/files#diff-f589ded5548e89f8e7153569e86178c754a3644b9c06d75dca1af23034c1b390R46